README.md

```diff
@@ -32,6 +32,44 @@ using the simple python tool from the `save` directory. The loader is a simple
 shell script so that you don't need python installed when loading the
 dashboards.
 
+
+Create a new dashboard
+----------------------
+
+If you added support for a new protocol in Packetbeat or a module in
+Metricbeat, it would be nice to create a dedicated Kibana dashboard to 
+prove the functionality of your data.
+The Kibana dashboards are saved in a special index in Elasticsearch. By default
+it's .kibana, but it can be configured to anything else.
+
+The first step, in creating your own Kibana dashboard, would be to load 
```

Comment:
The first step in creating your own Kibana dashboard is to load all the current dashboards into Kibana by using the
`load.sh` script. [you might also want to add a link to the doc about running the script?]